src/modules/welcome/module.js

```diff
@@ -5,7 +5,10 @@ import { getChannel, sanitizeUserInput } from '../../util';
 
 const subscriptions = new Map();
 
-subscriptions.set('guildMemberAdd', async (client, member) => {
+subscriptions.set('guildMemberUpdate', async (client, oldMember, member) => {
+    // checks if the pending status (which is true while screening is still in progress) has changed.
+    // as it can only be set to false/null afterwards, this should work fine.
+    if (oldMember.pending === member.pending) return;
     if (member.guild.id !== config.defaultGuild) return;
     await member.send(welcomeMessage).catch((e) => console.error('Error sending dm', e));
```

Comment:
Wouldn't we want to send the message right away anyways? This also prevents the bot from not sending the message if the bot was down when a user verfied.

Reply:
you can say the same for the system right now, as when the bot is down when a user joins we don't send a message either

Reply:
and people who don't accept the rules aren't welcome anyways on the server. 
in addition to that, screening gets skipped when we add roles

Reply:
I guess yeah, the message isn't sent if the bots down now. However, your second argument doesn't work, as I was just talking about just the message. I'd say the message is fine, as it does tell the user about the rules too,